x-pack/plugins/metrics_entities/server/services/stop_transforms.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+// TODO: Write this
```

Comment:
For now these must be stopped manually via the Transform UI/API correct?

Reply:
Yes, that is correct for right now. This part is not written.